person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1...1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+
+  attr_accessor :name, :age
+
+  def of_age?
+    @age >= 18
+  end
+
```

Comment:
- [ ] As it is a [requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md) for this project, " `Private method is_of_age? that returns true if @age is greater or equal to 18 and false otherwise.` ", please ensure your `of_age?` method is private, and it will also be nice to call within  the `can_use_services? method. Something like so: 

```ruby
  def can_use_services? 
    of_age?() || @parent_permission
  end
```
```
  private 
  def of_age?
    @age >= 18
  end
```
Please kindly fix that.

Reply:
@Mihndim2020 Thank you very much for your kind review and feedback. The functionality for this method is the same as the requirements. Cause linters show errors for writing if-else statements that return true and false and we should use "if else iteration" and write it like this.